src/components/PathwaysList/NewPathwayModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useCallback, useRef, memo, FormEvent } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faTimes } from '@fortawesome/free-solid-svg-icons';
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+  IconButton
+} from '@material-ui/core';
+import { v4 as uuidv4 } from 'uuid';
+
+import { usePathwayContext } from '../PathwayProvider';
+import useStyles from './styles';
+
+interface NewPathwayModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const NewPathwayModal: FC<NewPathwayModalProps> = ({ open, onClose }) => {
+  const styles = useStyles();
+  const pathwayNameRef = useRef<HTMLInputElement>(null);
+  const pathwayDescRef = useRef<HTMLInputElement>(null);
+  const { addPathway } = usePathwayContext();
+
+  const createNewPathway = useCallback(
+    (event: FormEvent<HTMLFormElement>): void => {
+      event.preventDefault();
+
+      addPathway({
```

Comment:
I noticed this no longer goes directly to the BuilderView. I feel like once a user creates a new pathway they would always want to begin building it. But now after creating it they have to select edit to open the builder view

Reply:
You are correct! Fixed.